diff --git a/org.fritzing.Fritzing.desktop b/org.fritzing.Fritzing.desktop
index e7899f90..56ab020d 100644
--- a/org.fritzing.Fritzing.desktop
+++ b/org.fritzing.Fritzing.desktop
@@ -14,3 +14,4 @@ Categories=Development;IDE;Electronics;
 Keywords=EDA;PCB;prototype;circuit;schematic;
 StartupNotify=true
 MimeType=application/x-fritzing-fz;application/x-fritzing-fzz;application/x-fritzing-fzp;application/x-fritzing-fzpz;application/x-fritzing-fzb;application/x-fritzing-fzbz;application/x-fritzing-fzm;
+StartupWMClass=Fritzing
