diff --git a/pri/svgppdetect.pri b/pri/svgppdetect.pri
index 017d8a10..736214d4 100644
--- a/pri/svgppdetect.pri
+++ b/pri/svgppdetect.pri
@@ -2,10 +2,10 @@

 message("Using fritzing svgpp detect script.")

-exists($$absolute_path($$PWD/../../svgpp-1.3.0)) {
-            SVGPPPATH = $$absolute_path($$PWD/../../svgpp-1.3.0)
+exists($$absolute_path($$PWD/../svgpp)) {
+            SVGPPPATH = $$absolute_path($$PWD/../svgpp)
             message("found svgpp in $${SVGPPPATH}")
         }

 message("including $$absolute_path($${SVGPPPATH}/include)")
